stateoftheuniverse/widgets/exoplanet_count/exoplanet_count.py

```diff
@@ -0,0 +1,127 @@
+"""
+Get the current number of exoplanets (grouped by different categories).
+"""
+
+# -----------------------------------------------------------------------------
+# IMPORTS
+# -----------------------------------------------------------------------------
+
+from collections import Counter
+
+from astroquery.nasa_exoplanet_archive import NasaExoplanetArchive
+
+
+# -----------------------------------------------------------------------------
+# FUNCTION DEFINITIONS
+# -----------------------------------------------------------------------------
+
+def get_exoplanet_class(mass: float,
+                        radius: float,
+                        units: str = 'Jupiter') -> str:
+    """
+    (Approximately) classify an exoplanet based on its mass and radius.
+
+    The classification scheme used here is based on the table from:
+        http://phl.upr.edu/library/notes/
+            amassclassificationforbothsolarandextrasolarplanets
+
+    Args:
+        mass: The mass of the exoplanet, in units of `unit` (i.e., if
+            `unit` is set to "Jupiter", the mass is assumed to be in
+            Jupiter masses).
+        radius: The radius of the exoplanet, in units of `unit` (i.e.,
+            if `unit` is set to "Jupiter", the radius is assumed to be
+            in Jupiter radii).
+        units: The unit system used for the `mass` and the `radius`.
+            Needs to be either "Earth" or "Jupiter".
+
+    Returns:
+        The approximate planet class, which is one of the following:
+            ["Asteroidan", "Mercurian", "Subterran", "Terran",
+             "Superterran", "Neptunian", "Jovian", "N/A", "Other"]
+    """
+
+    # Convert from Jupiter units to Earth units, if necessary
+    if units == 'Jupiter':
+        mass *= 317.907
```

Comment:
For consistency we should use the `astropy.units` module for these conversions. I can help with this if you need.

Reply:
Actually starting to think that all the stuff that relies on units here should be astropy unit compatible cos that would be super neat :D

Reply:
Sounds like a good idea! I guess we should all briefly coordinate about this, though, to make sure it's implemented consistently across all widgets?

Reply:
We decided that if something has a unit, it should have an astropy unit.